packages/utils/recursive-parser.js

```diff
@@ -39,6 +39,9 @@ module.exports = function recursiveTransform(j, ast, key, value, action) {
 			if (action === "add") {
 				// update property/replace
 				j(p).replaceWith(utils.addProperty(j, p, key, value, action));
+			} else if (action === "remove") {
+				// remove property
+				j(p).remove();
```

Comment:
is this enough?

Reply:
Definitely not enough. Working on AST parser for removal..

Reply:
How is it going? What have you done today?